Tests/NIOPosixTests/AsyncChannelBootstrapTests.swift

```diff
@@ -586,8 +588,108 @@ final class AsyncChannelBootstrapTests: XCTestCase {
             }
         }
     }
+    
+    // MARK: - Pipe Bootstrap
+    
+    func testPipeBootstrap() throws {
+        XCTAsyncTest {
```

Comment:
I'm pretty sure we no longer need `XCTAsyncTest`, `async` tests should work from 5.6